pennylane/transforms/split_qscript.py

```diff
@@ -338,28 +229,33 @@ def sum_expand(qscript: QuantumScript, group=True):
                 tmp_idxs.append([idxs_coeffs[measurements.index(m)] for m in m_group])
         idxs_coeffs = tmp_idxs
         qscripts = [
-            QuantumScript(ops=qscript._ops, measurements=m_group, prep=qscript._prep)
+            qscript.__class__(ops=qscript._ops, measurements=m_group, prep=qscript._prep)
```

Comment:
What motivated this change? As QuantumScript is imported anyway

Reply:
When using `QuantumScript` some tests failed.

I changed it to `QuantumTape`.